internal/api/api.go

```diff
@@ -61,6 +66,8 @@ type status struct {
 
 // NewIdlerAPI creates a new instance of IdlerAPI.
 func NewIdlerAPI(userIdlers *openshift.UserIdlerMap, clusterView cluster.View) IdlerAPI {
+	// Initialize metrics
+	Recorder.Initialize()
```

Comment:
FYI, you need to call Initialize() method only once in lifecycle of your service.  Initialize() call registerMetrics(), as registerMetrics() is sort of idempotent, calling it multiple time will not be an issue.  This comment is just for your attention.

Reply:
I "think" that `NewIdlerAPI` is called just once.